share/spack/gitlab/cloud_pipelines/stacks/gcp-base/spack.yaml

```
@@ -0,0 +1,29 @@
+spack:
+  view: false
+  packages:
+    cmake:
+      version:
+      - 3.27.9
+  definitions:
+  - arch:
+    - "%gcc"
+
+  - packages:
+    - cmake
+
+  - targets:
+    - "target=x86_64_v3"
+
+  specs:
+  - matrix:
+    - - $packages
+    - - $arch
+    - - $targets
+  ci:
+    pipeline-gen:
+    - build-job:
+        tags: ["gcp"]
```

because we re tagged the runners, this won't quite work. We need to remove the non-gcp `<arch>` tags and add the `<arch>-gcp` tags for the GCP runners. I should have been more clear about where all we needed to retag before, sorry about that.

```suggestion
        tags: ["gcp", 'x86_64-gcp"]
      build-job-remove:
        tags: ["x86_64", "x86_64_v2", "x86_64_v3", "x86_64_v4"]
```

---

Thanks!